composer.json

```diff
@@ -77,5 +78,7 @@
             "cakephp/plugin-installer": true,
             "dealerdirect/phpcodesniffer-composer-installer": true
         }
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
Is this needed with explicit versions?

Reply:
Yes it needed to be able to install indirect non-stable dependencies. In this particular case `bake` has `twig-view` with `dev-cake5`.

Reply:
Ah right.